### DISCUSSION TITLE
Block editor layout not present in post editor

### DISCUSSION THREAD

**Discussion by SWPeats** [WordPress/gutenberg#54955] [Q&A]
I have a WordPress block theme entirely built using the block editor, i.e. empty html template files. Since a recent update of WordPress, the post editor is not abiding to the width applied in the theme editor. In the theme editor (Code editor) I can see that `"layout":{"type":"constrained","contentSize":"1135px"}` is present and working correctly, however in the post editor, the class `is-layout-flow` has been applied, rather than `is-layout-constrained`. What am I missing?

I have cleared the browser and website cache, re-saved the theme, and disabled all plugins.

**Comment by unscripted** [comment]
I'm having this exact issue. Did you find a solution to this?

**Comment by unscripted** [comment]
It turns out I had a template revision saved in the database, so my code edits to the HTML template locally were not being used.

I then pulled the template back into the theme using the Create Block Theme plugin.

If you try to fix this in the editor, you need to select the "Content" block and enable "Inner blocks use content width."

<img width="277" alt="Screenshot 2024-09-29 at 5 09 51 PM" src="https://github.com/user-attachments/assets/48e1ebdf-e554-4b5d-8ecc-987c11c094ef">
